Wiki/DefaultConfig/index.md

```diff
@@ -3,6 +3,123 @@ layout : wiki
 title : DefaultConfig
 ---
 
+At the end of this overview, you find the current default configuration 
+Fvwm ships with. It is obvious that such a default setting needs to make
+choices, and this page tries to explain some of these choices so as to 
+enable the (new) user to understand them and to begin adapting the config
+to their own needs. Again, this needs to be limited, please refer to the 
+man page for a more exhaustive discussion of all available options.
+
+1: Functions
+2: Styles
+3: Colorsets
+4: Menus
+5: Bindings
+6: Decor
+7: Modules
+
+
+# Functions
+
+There is already a more elaborate page on functions:  
+<a href="/wiki/Config/Functions">Functions</a>
+
+The StartFunction can also be used an elegant autostart section.
+You can, e.g., add
+
+{% highlight fvwm %}
++ I Test (Init) exec conky
+{% endhighlight %}
+
+to start your standard conky right at startup.
+
+# Styles
+
+Styles are settings that, broadly speaking, influence everything that is
+related to windows: their looks ("decor"), their behaviour and the way
```

Comment:
I would caution against saying that Styles influence looks.  Decors are very much separate, although there's an annoying crossover with things like Colorsets being set in Styles, looks are still largely set in Decors.

Reply:
"Annoying crossover". :D 
Again, I was trying to put things as easily as possible, which was too much, here. I try again, but I might run into problems because I might not have got the underlying basics correctly myself, so please expect this point to develop more slowly than others.